package.json

```diff
@@ -73,5 +78,28 @@
     "node": ">= 6.9.1",
     "npm": ">= 3.10.3",
     "yarn": ">= 0.20.1"
+  },
+  "jest": {
+    "automock": true,
+    "verbose": true,
+    "collectCoverage": true,
+    "collectCoverageFrom": [
+      "app/javascript/packs/**/*.js"
+    ],
+    "coverageReporters": [
+      "lcov"
+    ],
+    "moduleNameMapper": {
+      "^.+\\.(png|gif|css|scss)$": "identity-obj-proxy"
+    },
+    "globals": {
+      "__testing__": true
+    },
+    "transform": {
+      "^.+\\.js$": "babel-jest"
+    },
+    "roots": [
+      "app/javascript/packs"
+    ]
```

Comment:
I'd explicitly specify which files should be picked up for testing, something like:

```
testMatch: ["**/*.test.js"]
```